packages/components/src/components/contract-card/contract-card-items/contract-card-header.tsx

```diff
@@ -7,6 +7,7 @@ import {
     getGrowthRatePercentage,
     isBot,
     isAccumulatorContract,
+    isOnlyUpsDownsContract,
```

Comment:
Currently we are enabling only one contract type from SmartTrader (`Only ups/ downs`). When there will be more contracts and they will all have common style, then I need to rename variable, something like `isOnlyUpsDownsContract` ---> `isSmartTraderContract` + refactor function which check if it is a SmartTrader contract.